packages/studio-web/src/app/ras.service.ts

```diff
@@ -46,41 +45,16 @@ export enum SupportedOutputs {
 })
 export class RasService {
   baseURL = environment.apiBaseURL;
-  constructor(private http: HttpClient, private toastr: ToastrService,) {}
+  constructor(private http: HttpClient) {}
 
-  convertRasFormat$(body: ReadAlongFormatRequest, output_type: SupportedOutputs|string): Observable<any|HttpErrorResponse> {
-    return this.http.post(this.baseURL + "/convert_alignment/" + output_type, body, {responseType: 'blob'}).pipe(
-      catchError((err: HttpErrorResponse) => { this.toastr.error(
-        err.message,
-        $localize`Hmm, we can't connect to the ReadAlongs API. Please try again later.`,
-        {
-          timeOut: 60000,
-        }
-      ); return of(err)} ),
-      take(1)
-    );
+  convertRasFormat$(body: ReadAlongFormatRequest, output_type: SupportedOutputs|string): Observable<any> {
+      return this.http.post(this.baseURL + "/convert_alignment/" + output_type, body, {responseType: 'blob'});
   }
 
-  assembleReadalong$(body: ReadAlongRequest): Observable<ReadAlong|HttpErrorResponse> {
-    return this.http.post<ReadAlong>(this.baseURL + "/assemble", body).pipe(
-      catchError((err: HttpErrorResponse) => { this.toastr.error(
-        err.message,
-        $localize`Hmm, we can't connect to the ReadAlongs API. Please try again later.`,
-        {
-          timeOut: 60000,
-        }
-      ); return of(err)} )
-    );
+  assembleReadalong$(body: ReadAlongRequest): Observable<ReadAlong> {
+      return this.http.post<ReadAlong>(this.baseURL + "/assemble", body);
   }
-  getLangs$(): Observable<Array<SupportedLanguage>|HttpErrorResponse> {
-    return this.http.get<Array<SupportedLanguage>>(this.baseURL + "/langs").pipe(
-      catchError((err: HttpErrorResponse) => { this.toastr.error(
-        err.message,
-        $localize`Hmm, we can't connect to the ReadAlongs API. Please try again later.`,
```

Comment:
this wasn't very DRY, by the way, duplicating the same error reporting twice just lines apart